js/jquery.fileupload-video.js

```diff
@@ -25,7 +25,8 @@
         // Node/CommonJS:
         factory(
             require('jquery'),
-            require('load-image')
+            require('blueimp-load-image/js/load-image'),
+            require('blueimp-load-image/js/load-image-meta')
```

Comment:
This dependency is not necessary. The loadImage dependency is only included for the cross-browser `createObjectURL` function, which might not even be necessary anymore today, but better to include for now.